### ISSUE TITLE
Unable to install with Python 3.6

### ISSUE THREAD

**Issue by dreerr** [plotdevice/plotdevice#47]
When trying to install PlotDevice with Python 3.6 the build.py complains:
```
    http://plotdevice.io/wheelhouse/pyobjc-3.0.4-py36.tar.gz
    curl: (22) The requested URL returned error: 404 Not Found
```
When trying to access the URL in the browser it does also throw a 404 error. Any idea?

**Comment by MrJanssen** [comment]
I get the same error.
Would love to use the plotdevice-module.

Mac OS Mojave 10.14.2
Python 3.7.2

**Comment by samizdatco** [comment]
fixed in 289893482b0900f3fc754c79e71a92d421883888
